packages/botonic-plugin-google-analytics/src/index.js

```diff
@@ -0,0 +1,75 @@
+import Analytics from 'analytics'
+import googleAnalytics from '@analytics/google-analytics'
+
+export default class BotonicPluginGoogleAnalytics {
+  /**
+   * @param {Object} options - Options for the plugin
+   * @param {string} options.trackingId - Tracking ID for Google Analytics.
+   * @param {function({session: object}): string} [options.userId] - Method that returns a unique user ID as string.
+   * @param {function({session: object}): object} [options.userTraits] - Method that returns the user traits as object.
+   * @param {boolean} [options.automaticTracking] - If set to false, no automatic tracking will be done in post method.
+   * @param {function({session: object, input: object, lastRoutePath: string}): object} [options.eventFields] - Method
+   *    that returns the eventFields to track as object (used only if automaticTracking is set to false).
```

Comment:
you meant "if automaticTracking is set to TRUE", right?

Reply:
yes! Good eyes :eyes: ! Corrected.